index.js

```diff
@@ -49,6 +49,10 @@ function copy(text, options) {
         e.clipboardData.clearData();
         e.clipboardData.setData(options.format, text);
       }
+      if (options.onCopy) {
+        e.preventDefault();
+        options.onCopy(e.clipboardData);
```

Comment:
I wonder whether logic better should be 
1) either provide onCopy callback that can do everything this cb does (stopPropagation/preventDefault clearData / change data) 
2) if no `onCopy` is provided, but there's `format` option, run this callback

I did forget why we needed original `stopPropagation` that would be called in any case, whether there's format specified or not 🤔 

Other possibility in terms of API would be to pass something more elaborate to copy fn with some sort of mapping `{[format]: content}` but this might be a breaking change and would need another checks during handling

Reply:
Personally I like having the original callback to handle the event (clearData, setData...)
As a lib user that takes the burden off my shoulders, so the new onCopy callback encapsulates only my business logic.
Do you think the code should change?

Reply:
No, I guess it's okay for time being, but I would probably plan to do the change with first publishing a lib that would warn when format is passed and then publish a new major with only onCopy callback called